qbx_core.sql

```diff
@@ -45,10 +45,10 @@ CREATE TABLE IF NOT EXISTS `player_contacts` (
 ) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
 
 CREATE TABLE IF NOT EXISTS `player_groups` (
-	`citizenid` VARCHAR(50) NOT NULL,
-	`group` VARCHAR(50) NOT NULL,
-	`type` VARCHAR(50) NOT NULL,
-	`grade` TINYINT(3) UNSIGNED NOT NULL,
-	PRIMARY KEY (`citizenid`, `type`, `group`),
-	CONSTRAINT `fk_citizenid` FOREIGN KEY (`citizenid`) REFERENCES `players` (`citizenid`) ON UPDATE CASCADE ON DELETE CASCADE
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
```

Comment:
If the default collation is already utf8mb4_unicode_ci, won't that apply to all columns already?

Reply:
> If the default collation is already utf8mb4_unicode_ci, won't that apply to all columns already?

Worked perfectly for me only does the one column